docs/home/howto/SECURITY.md

```diff
@@ -367,3 +367,257 @@ cp /etc/kubernetes/admin.conf $HOME/
 chown $(id -u):$(id -g) $HOME/admin.conf
 export KUBECONFIG=$HOME/admin.conf
 ```
+
+## How to turn on Hashicorp Vault functionality
+
+In Epiphany beside storing secrets in Kubernetes secrets there is also a possibility of using secrets stored in Vault
+from Hashicorp. This can provide much more sophisticated solution for using secrets and also higher level of security
+than standard Kubernetes secrets implementation. Also Epiphany provides transparent method to access Hashicorp Vault
+secrets with applications running on Kubernetes. You can read in the more about it in [How to turn on Hashicorp Vault integration with k8s](./SECURITY.md#how-to-turn-on-hashicorp-vault-integration-with-k8s) section. In the future we want also to provide additional features
+that right now can be configured manually according to Hashicorp Vault [documentation](https://www.vaultproject.io/docs).
+
+At the moment only installation on Kubernetes Master is supported, but we are also planning separate installation with no
+other components. Also at this moment we are not providing clustered option for Vault deployment, but this will be part
+of the future releases.
+
+Below you can find sample configuration for Vault with description of all options.
+
+```yaml
+kind: configuration/vault
+title: Vault Config
+name: default
+specification:
+  vault_enabled: true # enable Vault install
+  vault_system_user: vault # user name under which Vault service will be running
+  vault_system_group: vault # group name under which Vault service will be running
+  enable_vault_audit_logs: false # turn on audit logs that can be found at /opt/vault/logs/vault_audit.log
+  enable_vault_ui: false # enable Vault UI, shouldn't be used at production
+  vault_script_autounseal: true # enable automatic unseal vault at the start of the service, shouldn't be used at production
+  vault_script_autoconfiguration: true # enable automatic configuration of Hashicorp Vault
```

Comment:
I believe that should be explained more. What auto configuration is doing? What is being configured?